views/js/controller/creator/helpers/qtiTest.js

```diff
@@ -82,13 +82,29 @@ define(['lodash', 'i18n'], function(_, __){
             };
         },
 
+        /**
+         * Gives you a validator that check QTI id format of the test (it is different from the others...)
+         * @returns {Object} the validator
+         */
+        testidFormatValidator : function testidFormatValidator(){
+            var qtiTestIdPattern = /^\S+$/;
```

Comment:
I've changed the rule in order to conform to the xsd validation. Then it's a bug that need to be fixed on qtism.